src/service.rs

```diff
@@ -0,0 +1,48 @@
+use Body;
+
+use http::{Request, Response};
+use futures::{Future, Poll};
+use tower::Service;
+
+/// An HTTP (2.0) service that backs the gRPC client
```

Comment:
what does `tower-h2` have to do with "the gRPC client"?

Reply:
Woops, it doesn't :) I didn't proof read comments.